Ginger/GingerCoreCommon/UIElement/ElementInfo.cs

```diff
@@ -259,6 +260,152 @@ public virtual string GetValue()
             return mValue;
         }
 
+
+
+
+
+
+
+        private bool mIsSelected = false;
+        public bool IsSelected
+        {
+            get
+            {
+                return mIsSelected;
+            }
+            set
+            {
+                if (mIsSelected != value)
+                {
+                    mIsSelected = value;
+                    OnPropertyChanged(nameof(IsSelected));
+                }
+            }
+        }
+
+
+
+        public enum eElementGroup
+        {
+            All,
+            Mapped,
+            Unmapped
+        }
+
+        public eElementGroup ElementGroup { get; set; }
+
```

Comment:
Mapped/Unmapped belong only to POM, ElementInfo is been used all a cross. please remove eElementGroup from here. you can have generic object "Group" field and in your POM relevant class you will set it with enum eElementGroup which will come from ApplicationPOMModel.cs

Reply:
better to create that field under HTMLElementInfo.